cdbot/cogs/cyber.py

```diff
@@ -167,10 +167,20 @@ async def flag(
             content = "13.1 is a No Flag Zone™ 🙅⛔⚔️"
         else:
             # Generates random, but unique and identical per challenge, base 64 "flag"
-            content = (
-                "The flag is:"
-                f"||{await generatebase64(ord(base[0]) + level_num + challenge_num)}||"
-            )
+            if random.randint(1, 5) == 5:
+
+                await ctx.send(
```

Comment:
the call to return and await can be combined.
```suggestion
                return await ctx.send(
```